person.rb

```diff
@@ -1,21 +1,25 @@
 require './nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
+
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
     super()
   end
 
   def correct_name
     @name
   end
 
-  attr_accessor :name, :age
-
-  attr_reader :id
+  def add_rental(book)
+    @rentals.push(book)
+  end
```

Comment:
- [ ] Kindly update this method also based on the aforementioned feedback in the book class. It should take a book and date as parameters and use it to create a rental

  ```
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
  ```

Reply:
Thanks for the explanation. The methods have been updated accordingly.